### ISSUE TITLE
Embedding UICaputre Video

### ISSUE THREAD

**Issue by marleen-linares** [ITS-UofIowa/international.uiowa.edu#39]
Hi Joe,
Hope you're well. I'm just wondering if there is an appropriate way to embed UICapture videos on webpages, other than switching to Full HTML? 

https://icon.uicapture.uiowa.edu/Panopto/Pages/Viewer.aspx?tid=eb71a3b2-46c8-48fb-abce-49b924ac81b9

Thanks,
Marleen

**Comment by joewhitsitt** [comment]
All is well, thanks!

I am not aware of a UICapture embed option and our embed media from URL doesn't support UICapture.

If you have embed code, you can send it my way and I will take a look. Most likely it will be the full html route, and going forward, I or another developer will have to place it for you.

**Comment by marleen-linares** [comment]
Thanks Joe! I've included the embed code below, although the file still requires access when embedded which is what we were trying to avoid. We're checking in with ITS to see if there's a way to download the file so then we can upload to YouTube and get it on the website.

<iframe src="https://icon.uicapture.uiowa.edu/Panopto/Pages/Embed.aspx?id=26982364-688f-4b7f-a449-ca47fd9321cd&v=1" width="720" height="405" style="padding: 0px; border: 1px solid #464646;" frameborder="0" allowfullscreen></iframe>

**Comment by joewhitsitt** [comment]
Yes, that iframe object will require developer placement in full_html going forward. If you can get it in another provider (youtube, vimeo, sharestream), those don't require a developer to place.
